src/components/file/file.repository.ts

```diff
@@ -93,20 +93,18 @@ export class FileRepository {
           createdBy: [{ id: 'createdById' }],
         },
       ])
-      .orderBy([`${input.sort} ${input.order}`])
-      .skip((input.page - 1) * input.count)
-      .limit(input.count);
-
+      .orderBy([`${input?.sort ?? 'name'} ${input?.order ?? 'ASC'}`])
```

Comment:
This is ordering by the name _node_. You want to order by the name _value_, if `sort` is _name_.

I did say order didn't on slack. But now I'm thinking we have to at least have one order so results are consistent back otherwise index-based pagination won't work.